modules/ztp-install-factory-spoke-pipeline.adoc

```diff
@@ -0,0 +1,186 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: PROCEDURE
+[id="run-the-edge-cluster_factory_install_pipeline_{context}"]
+= Run the edge cluster factory install pipeline
+
+The edge factory install pipeline is as follows.
+
+.Prerequisites
+
+* Enough DHCP IPs in the external and factory network to hold the edge cluster.
```

Comment:
What is factory network here? Is it the isolated one? We set IPs using nmstate on it.

Reply:
term factory not needed here